README.md

```diff
@@ -119,6 +119,14 @@ BUILD_NUMBER=12
 ### Custom CSS
 We have provided a sample CSS in ./static/css/oscar-custom.css. Feel free to play with this.
 
+### Customizing OpenOsp Properties
+1. After starting your container `./start.sh`
+2. You should be able to go to `/properties` and login. You can edit credentials on your`local.env` file:
+  * By default, it should be openosp and openosp
+  * `FLASK_USERNAME`
+  * `FLASK_PASSWORD`
```

Comment:
Please disable this feature by default instead so someone doesn't accidentally allow it.